ui/src/hooks/useFileManage.ts

```diff
@@ -438,6 +452,8 @@ const handleFileUpload = (
 
           if (percent === 100) {
             onFinish();
+
+            mittBus.emit('reload-table');
           }
         }
       } catch (e) {
```

Comment:
This code has a difference between two strings:
`messageData = [...messageData];`
and
`if (fileManageStore.currentPath === '/' && typeof fileManageStore.fileSystem !== undefined);`

Optimization suggestion:
Add a `return` statement after the condition checking.

```js
const handleSocketSftpData = (messageData: IFileManageSftpFileItem[]) => {  
  fileManageStore.setFileList(messageData);
};
```

Since this function only returns nothing if current path equals '/', it's unnecessary to add another check here.